.github/workflows/frontend_checks.yml

```diff
@@ -73,6 +73,12 @@ jobs:
           wait-on: "http://localhost:3000"
           wait-on-timeout: 120
 
+      - uses: actions/upload-artifact@v3
```

Comment:
@NevilleS - This uploads a <10mb zip on test failure. Based on the GH docs I don't think this pushes any data limits, but I figured I'd call it out.

Reply:
where does this get uploaded to? also, will it upload all failing tests, or just the first one? (if all, then the zip could potentially be larger?)

Reply:
They live in the github ether and show up in the action summary:
![Screen Shot 2022-11-10 at 10 36 55](https://user-images.githubusercontent.com/2236777/201178953-f0886263-2fb2-46cb-a993-692d7b0d9121.png)

That 5.7mb was from a single spec failing, but has all the recordings. I'm sure we can make it smarter if we need to.